com.gitlab.news_flash.json

```diff
@@ -0,0 +1,39 @@
+{
+    "app-id" : "com.gitlab.news_flash",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "com.gitlab.newsflash",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--device=dri",
+        "--talk-name=org.freedesktop.Notifications"
```

Comment:
do you really need this D-Bus name? Gtk handle notifications properly without it.

Reply:
I didn't know that. Thanks!